tests/test_two_sided_market.py

```diff
@@ -437,7 +450,8 @@ def test_match_recommendations(self, market):
         recommendations = [
             BidOfferMatch(
                 bids=[bid.serializable_dict()], offers=[offer.serializable_dict()],
-                trade_rate=2, selected_energy=1, market_id=market.id).serializable_dict()
+                trade_rate=2, selected_energy=1, market_id=market.id,
+                time_slot="time_slot").serializable_dict()
```

Comment:
Nit: For my taste it would be good to have a near-real timestamp here for the tests in order to show the code reader how a "normal" time stamp looks like:
```suggestion
                time_slot="20211006T12:00").serializable_dict()
```